src/core/io/src/4C_io_input_field.hpp

```diff
@@ -168,12 +168,18 @@ namespace Core::IO
     /**
      * Access the value of the field for the given @p element index. If the @p element_id
      * is not a valid index, this function will throw an error.
+     * @param element_id The index of the element for which to get the value.
+     * @param field_name Optional name of the field for error messages. This is particularly useful
+     * if multiple input fields are used.
```

Comment:
I am not against this checklist style per se, but in my experience, documentation for short functions reads a lot better if it is written in full sentences. So, how about you add after the original text: 


```suggestion
     * which contains the optional @p field_name for informational purposes.
```